### ISSUE TITLE
Understand Accessibility and the CMS User

### ISSUE THREAD

**Issue by joshworkscivicactions** [department-of-veterans-affairs/va.gov-cms#5754]
## Description
As Help Desk Manager I need to understand where accessibility (508 compliance) and the CMS user intersect so that I can - develop processes?  drive 508 compliance?

- Review Stan in Slack for conversations about accessibility issues 
- Define the types of content that is not compliant
- Get team input 

## Acceptance Criteria
- [ ] Help Desk has a process to manage 508 compliance when a CMS user is responsible for the content that is not compliant 

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [] `Core Application Team`
- [x] `Product Support Team`
